### PR TITLE
Check code coverage of every package

### PR DESCRIPTION
**Pull request by yutongz**
https://github.com/istio/istio-testing/issues/135

Add a script to check if code coverages of specific packages meet requirements.
Requirements are specified in "codecov.requirement"
Rely on external dependency from istio/toolbox, which is shared through other repos.

---
This change is [Reviewable](https://reviewable.io/reviews/istio/mixer/506)

### CODE REVIEW

File: WORKSPACE [hunk 1]
```diff
@@ -18,6 +18,12 @@ git_repository(
 
 load("@org_pubref_rules_protobuf//protobuf:rules.bzl", "proto_repositories")
 
+git_repository(
+    name = "istio_toolbox",
+    remote = "https://github.com/istio/toolbox.git",
+    commit = "55ee1cfbb22863051a45f084404fdf9a77046fc4"
```
Comment (douglas-reid) [inline WORKSPACE]: please add a date and release annotation comment here (to match the others in the file).

File: codecov.requirement [hunk 2]
```diff
@@ -0,0 +1,5 @@
+Requirements of code coverage percentage for packages
+Use this format: "Package name \t Required percentage"
+istio.io/mixer/adapter/denyChecker	99
```
Comment (douglas-reid) [inline codecov.requirement]: just curious: who will be responsible for adding the rest of the packages?

Comment (yutongz) [inline codecov.requirement]: It should be maintained by dev teams i.e. you guys. You can set the thresholds for your pace. What here is just a example showing what format it's supposed to be.

Comment (douglas-reid) [inline codecov.requirement]: SGTM.

**Comment by yutongz** [comment]
This pr is ready for merge.

**Comment by douglas-reid** [comment]
Needs an update (out of date with branch). Then I'll merge.

File: WORKSPACE [hunk 3]
```diff
@@ -268,3 +268,14 @@ new_go_repository(
...
 )
+
+
+##
+## Testing
+##
+
+git_repository(
+    name = "istio_test_infra",
+    commit = "983183f98b79f8b67fe380fef4cdd21481830fd7",
```
Comment (douglas-reid) [inline WORKSPACE]: we just need to annotate this commit with a comment including date and version info.

Comment (yutongz) [inline WORKSPACE]: I just noticed that, added.

**Comment by douglas-reid** [comment]
Sorry, needs another update.

**Comment by yutongz** [comment]
https://github.com/istio/toolbox/issues/2

**Pull request merged**